api-client/src/utils/eventExport.ts

```diff
@@ -0,0 +1,45 @@
+import moment, { Moment } from 'moment'
```

Comment:
I think the contents of this file should perhaps be moved to `EventModel` to a method called `toICal` or something. At least that would be the place I would look first for it. Other opinions welcome as well though.